thirtyinch/src/test/java/net/grandcentrix/thirtyinch/internal/TestTiFragment.java

```diff
@@ -19,16 +19,15 @@
 import static org.mockito.Mockito.*;
 
 import android.os.Bundle;
-import android.support.annotation.NonNull;
 import android.view.LayoutInflater;
 import android.view.ViewGroup;
+import androidx.annotation.NonNull;
 import java.util.concurrent.Executor;
 import net.grandcentrix.thirtyinch.TiPresenter;
 import net.grandcentrix.thirtyinch.TiView;
 
 
 /**
- * mock implementation of a {@link android.support.v4.app.Fragment} with all relevant lifecycle
```

Comment:
why removed?

Reply:
Sorry, added again

Reply:
I don't know why that line is still visible here because it's added actually.